### DISCUSSION TITLE
How can I import a VM

### DISCUSSION THREAD

**Discussion by mondoshivan** [utmapp/UTM#6433] [General]
We have used vagrant and VMWare Fusion in our continuous integration system to run installer based tests inside Mac and Windows VMs. Our content is quiet large and we use multiple VMs with different operating systems therefore we are using a self hosted centralised vm-share in our company, which archives and distributes all of our VMs. We are looking for a VMWare alternative and UTM seems to be promising. However there seems to be no option to import UTM VMs or did I miss this feature? Preferably we would like to use the UTM CLI for importing our VMs. Is it possible or planned for the future?

**Comment by naveenrajm7** [comment]
Maybe this request is similar to https://github.com/utmapp/UTM/issues/6612 ?
